daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -13,6 +13,65 @@
 from openrazer_daemon.misc.key_event_management import NagaHexV2KeyManager as _NagaHexV2KeyManager
 
 
+class RazerLanceheadTE(__RazerDeviceSpecialBrightnessSuspend):
+    """
+    Class for the Razer Lancehead Tournament Edition (Wired)
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Lancehead_TE-if0(1|2)-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0060
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 16]
+    METHODS = ['get_device_type_mouse', 'set_key_row', 'max_dpi', 'get_dpi_xy', 'set_dpi_xy', 'get_poll_rate', 'set_poll_rate', 'set_idle_time', 'set_low_battery_threshold', 'get_battery', 'is_charging', 'get_logo_brightness', 'set_logo_brightness', 'get_scroll_brightness', 'set_scroll_brightness',
+               # Effects
+               # 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect', 'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect', 'set_custom_effect',
+               # Logo
+               'set_logo_wave_lancehead_te', 'set_logo_static_lancehead_te', 'set_logo_spectrum_lancehead_te', 'set_logo_none_lancehead_te', 'set_logo_reactive_lancehead_te', 'set_logo_breath_random_lancehead_te', 'set_logo_breath_single_lancehead_te', 'set_logo_breath_dual_lancehead_te',
+               # Scroll wheel
+               'set_scroll_wave_lancehead_te', 'set_scroll_static_lancehead_te', 'set_scroll_spectrum_lancehead_te', 'set_scroll_none_lancehead_te', 'set_scroll_reactive_lancehead_te', 'set_scroll_breath_random_lancehead_te', 'set_scroll_breath_single_lancehead_te', 'set_scroll_breath_dual_lancehead_te',
+               # Can set LOGO and Scrol with custom
+               'set_custom_effect', 'set_key_row']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1203/1206_lanceheadte.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
```

Comment:
Please remove this block

Reply:
Ok, so when I removed the entire RAZER_URLS block it broke the device image in RazerGenie.  If I leave the code block with only top_img the device image shows again.

Reply:
It has been removed.